src/main/resources/swagger.api/inn-reach.yaml

```diff
@@ -5,175 +5,161 @@ info:
         '500':
           description: Internal server error
           content:
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: Internal server error
-      description: Add new inn reach configuration
-      operationId: postConfiguration
+                $ref: "#/components/schemas/error"
+      description: Add new InnReach location
+      operationId: postInnReachLocation
       requestBody:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/moduleConfiguration"
+              $ref: "#/components/schemas/innReachLocationDTO"
         required: true
-    put:
+    get:
       responses:
-        '201':
-          description: ""
+        '200':
+          description: "OK"
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/moduleConfiguration"
-        '400':
-          description: Bad request
+                $ref: "#/components/schemas/innReachLocationsDTO"
+        '500':
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+      description: Get a list of InnReach locations
+      operationId: getLocations
+      parameters:
+        - $ref: "#/components/parameters/trait_pageable_offset"
+        - $ref: "#/components/parameters/trait_pageable_limit"
+  /locations/{locationId}:
+    get:
+      responses:
+        '200':
+          description: OK
           content:
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: Bad request
+                $ref: "#/components/schemas/innReachLocationDTO"
         '404':
-          description: Configuration not found
+          description: InnReach location not found
           content:
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: Configuration not found
+                $ref: "#/components/schemas/error"
         '500':
           description: Internal server error
           content:
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: Internal server error
-      description: Change the inn reach configuration
-      operationId: putConfiguration
-      requestBody:
-        content:
-          application/json:
-            schema:
-              $ref: "#/components/schemas/moduleConfiguration"
-        required: true
-  /configurations/{configId}:
-    get:
+                $ref: "#/components/schemas/error"
+      description: Get InnReach location by id
+      operationId: getLocationById
+      parameters:
+        - $ref: '#/components/parameters/locationId'
```

Comment:
Does it make sense to define common parameter for id which refers for UUID definition schema in "parameters" section and the use it everywhere when "identifiers" are faced? In contrast to duplicate id definition for every entity's identifier, of course if all there identifiers have the same nature, what do you think ?